### PR TITLE
✨Add ISCN status column on post list page

### PR DESCRIPTION
**Pull request by shellyYG**

### CODE REVIEW

File: likecoin/admin/editor.php [hunk 1]
```diff
@@ -76,6 +78,7 @@ function likecoin_add_posts_columns( $columns ) {
  * @param int|    $post_id The current post ID.
  */
 function likecoin_populate_posts_columns( $column, $post_id ) {
+	global $post;
```
Comment (williamchong) [inline likecoin/admin/editor.php:81]: Where is `$post` used?

File: likecoin/admin/editor.php [hunk 2]
```diff
@@ -95,12 +98,56 @@ function likecoin_populate_posts_columns( $column, $post_id ) {
 				echo esc_url( $status[ $column ]['url'] );
 				?>
 					">
-				<?php echo esc_html( $status[ $column ]['status'] ); ?>
+				<?php echo esc_html( $status[ $column ]['status'] ); ?>
```
Comment (williamchong) [inline likecoin/admin/editor.php:101]: Please remove extra space

File: likecoin/assets/js/admin/likecoin_metabox.js [hunk 3]
```diff
@@ -75,8 +75,14 @@ function onSubmitToISCN(e) {
 }
 
 (() => {
+  const submitISCNHash = window.location.hash;
   const refreshBtn = document.getElementById('lcPublishRefreshBtn');
   const submitISCNBtn = document.getElementById('lcISCNPublishBtn');
+  if (submitISCNHash === '#likecoin_submit_iscn') {
+    setTimeout(() => {
+      window.scrollTo(0, document.querySelector('#poststuff').scrollHeight);
```
Comment (williamchong) [inline likecoin/assets/js/admin/likecoin_metabox.js:83]: When is `#poststuff` defined? Can we define `#likecoin_submit_iscn` in our metabox instead?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: likecoin/admin/editor.php (hunk 1 above)
Comment (shellyYG) [reply likecoin/admin/editor.php:81]: here: $publish_params = likecoin_get_meta_box_publish_params( $post );

File: likecoin/assets/js/admin/likecoin_metabox.js (hunk 3 above)
Comment (shellyYG) [reply likecoin/assets/js/admin/likecoin_metabox.js:83]: I got that by using developer tool. But yes, I think the later is more specific!

**Review by williamchong — APPROVED** [review]

**Pull request merged**